pkg/plugins/loader.go

```diff
@@ -278,7 +287,7 @@ func (p *PluginLoader) LoadPlugins(ctx context.Context, pluginDir string, scheme
 		md := md
 		clientOpts := options.clientOptions
 		if verifyManifest {
-			if secureConfig, ok := secureConfigs[md.Module]; ok {
+			if secureConfig, ok := secureConfigs[path.Base(md.Module)]; ok {
```

Comment:
Plugins are identified by the complete module path, just the base name isn't enough

Reply:
That's not going to work in the new design because we don't have access to the full module path in the manifest; only the URN